api/soap/mantisconnect.wsdl

```diff
@@ -45,6 +45,7 @@
    <xsd:element name="account_data" type="tns:AccountData" minOccurs="0"/>
    <xsd:element name="access_level" type="xsd:integer" minOccurs="0"/>
    <xsd:element name="timezone" type="xsd:string" minOccurs="0"/>
+   <xsd:element name="api_token_type" type="xsd:integer" minOccurs="0"/>
```

Comment:
Yes, but crash reporting software exists already, and it supports Mantis with login/password and, probably, (full) login/API token. When crash reporting API will be added, this software has to be modified to support new possibilities. 

So, how could such software detect what type of Mantis credentials were entered by developer? Is it login/password? Is it login/full token? Is it login/restricted token?

I understand that it does not make much sense to use full token or even login/password pair to call a single crash API function. However, there are people out there who already use such software and they have their settings all set up. Not everyone will immediately make changes in their configuration (e.g. replace login/password and/or full token with restricted token) when upgrading to new Mantis and crash reporting software.